src/core/Microsoft.PowerToys.Settings.UI/ViewModels/FancyZonesViewModel.cs

```diff
@@ -20,21 +20,45 @@ public class FancyZonesViewModel : Observable
 
         public ButtonClickCommand LaunchEditorEventHandler { get; set; }
 
-        public ICommand SaveColorChoiceEventHandler
+        public ICommand SaveZoneHighlightColorEventHandler
         {
             get
             {
-                return new RelayCommand<Color>(SaveColorChoice);
+                return new RelayCommand<Color>(SaveZoneHighlightColor);
+            }
+        }
+
+        public ICommand SaveBorderColorEventHandler
+        {
+            get
+            {
+                return new RelayCommand<Color>(SaveZoneBorderColor);
+            }
+        }
+
+        public ICommand SaveInActiveColorEventHandler
+        {
+            get
+            {
+                return new RelayCommand<Color>(SaveZoneInActiveColor);
             }
         }
 
         private FancyZonesSettings Settings { get; set; }
 
         public FancyZonesViewModel()
         {
-            Settings = SettingsUtils.GetSettings<FancyZonesSettings>(ModuleName);
+            try
+            {
+                Settings = SettingsUtils.GetSettings<FancyZonesSettings>(ModuleName);
+            }
+            catch
+            {
+                Settings = new FancyZonesSettings();
+                SettingsUtils.SaveSettings(Settings.ToJsonString(), ModuleName);
```

Comment:
This can throw as well. Not in this PR but we should add the logic to catch and handle the `IOExceptions` in the `SaveSettings` or `GetSettings` since they will be shared by all the modules.

Reply:
I will do it in the SettingsUtils.SaveSettings itself